website/react-magma-docs/src/pages/design/badges.mdx

```diff
@@ -57,8 +57,6 @@ Counters can be used as part of links or buttons, or simply in a context that co
   <img src="../../images/badges/label-colors.png" alt=" " />
 </figure>
 
-There are multiple colors available for badges, but they should be used with purpose. The colors that should be used most commonly are primary, secondary, and light. The best color to use will most likely be determined by which one gives the appropriate level of emphasis with the surrounding content.
+There are multiple colors available for badges, but they should be used with purpose. The colors that should be used most commonly are primary, high contrast, and low contrast. The best color to use will most likely be determined by which one gives the appropriate level of emphasis with the surrounding content.
```

Comment:
For the purposes of not breaking the current API, I was thinking of keeping the names `secondary` and `light` for badges in my PR (https://github.com/cengage/react-magma/pull/672).
![image](https://user-images.githubusercontent.com/91160746/166539938-8884e2b2-dc26-46ec-be69-d45fd9967396.png)

Are you opposed to that?

Reply:
I can get on board with that.  Secondary and Light can be the technical names, and high and low contrast can reference more of how we refer to them or how we use them in conversation.

I'll make some updates to the docs for that.